custom/auth/client/utils/feeutils.go

```diff
@@ -144,29 +144,29 @@ func FilterMsgAndComputeTax(clientCtx client.Context, msgs ...sdk.Msg) (taxes sd
 
 			taxes = taxes.Add(tax...)
 
-		case *wasmtypes.MsgInstantiateContract:
-			tax, err := computeTax(clientCtx, taxRate, msg.Funds)
-			if err != nil {
-				return nil, err
-			}
+			/*case *wasmtypes.MsgInstantiateContract:
+				tax, err := computeTax(clientCtx, taxRate, msg.Funds)
+				if err != nil {
+					return nil, err
+				}
 
-			taxes = taxes.Add(tax...)
+				taxes = taxes.Add(tax...)
 
-		case *wasmtypes.MsgInstantiateContract2:
-			tax, err := computeTax(clientCtx, taxRate, msg.Funds)
-			if err != nil {
-				return nil, err
-			}
+			case *wasmtypes.MsgInstantiateContract2:
+				tax, err := computeTax(clientCtx, taxRate, msg.Funds)
+				if err != nil {
+					return nil, err
+				}
 
-			taxes = taxes.Add(tax...)
+				taxes = taxes.Add(tax...)
 
-		case *wasmtypes.MsgExecuteContract:
-			tax, err := computeTax(clientCtx, taxRate, msg.Funds)
-			if err != nil {
-				return nil, err
-			}
+			case *wasmtypes.MsgExecuteContract:
+				tax, err := computeTax(clientCtx, taxRate, msg.Funds)
+				if err != nil {
+					return nil, err
+				}
 
-			taxes = taxes.Add(tax...)
+				taxes = taxes.Add(tax...)*/
```

Comment:
i think we can just remove this to avoid confusion in the future. We can just have a note saying that it is `nonTaxableTaxes` for smartcontract, which will be handle by reverse charge and no need to calculate it in the CLI

Reply:
yes that is what i mean